Common/UnitDefinitions/Force.json

```diff
@@ -121,6 +121,18 @@
           "Abbreviations": [ "фунт-сила" ]
         }
       ]
+    },
+    {
+      "SingularName": "OunceForce",
+      "PluralName": "OuncesForce",
```

Comment:
Is plural form ounce-force (OunceForce)?
https://www.convertunits.com/from/ounces/to/ounce-force

Reply:
I may be loosing my mind 😆 fixed.